### PR TITLE
catch someone!

### PR DESCRIPTION
**Pull request by UhiOfficial**

### CODE REVIEW

File: axol/axol.py [hunk 1]
```diff
@@ -2093,6 +2093,23 @@ def love_mute(message):
         return True
 
 
+caught = False
+
+
+def catch(message):
+    if message.chat.id != vip_chat_id:
```
Comment (Deepwarrior) [inline axol/axol.py:2100]: vip_chat - old chat, where nobody writes

File: axol/axol.py [hunk 2]
```diff
@@ -2093,6 +2093,23 @@ def love_mute(message):
...
+        return
+    global caught
+    if caught:
```
Comment (Deepwarrior) [inline axol/axol.py:2103]: You could use next message as input with ` bot.register_next_step_handler`
See for example:
https://github.com/eternnoir/pyTelegramBotAPI/blob/888c7a6b0d97c376947ee7b525d4b70583acc3fc/examples/step_example.py

File: axol/axol.py [hunk 3]
```diff
@@ -2093,6 +2093,23 @@ def love_mute(message):
...
+        caught = False
+        bot.send_sticker(message.chat.id, random.choice(config.catching_hands))
+    else:
+        if random.randint(0, 1000) != 228:
+            return
+        caught = True
+        bot.send_sticker(message.chat.id, 'CAACAgIAAxkBAAIfjWDhr5dRAZlK_8xkJUEvrepeAW7_AAL_DAACtTNJSgfaJegf7oIIIAQ')
```
Comment (Deepwarrior) [inline axol/axol.py:2110]: I think we need some timeout for fail. And then podcleanitb sledy

**Review by Deepwarrior — CHANGES_REQUESTED** [review]

File: axol/axol.py (hunk 2 above)
Comment (UhiOfficial) [reply axol/axol.py:2103]: If we were on Stack Overflow, I'd upvote this advice.

File: axol/axol.py [hunk 4]
```diff
@@ -2093,6 +2095,31 @@ def love_mute(message):
...
+
+
+def gotcha(message, timer):
+    timer.cancel()
+    bot.send_sticker(message.chat.id, random.choice(config.catching_hands))
+
+
+def podcleanitb_sledy(message):
+    try:
+        bot.clear_step_handler(message)
```
Comment (Deepwarrior) [inline axol/axol.py:2117]: wow, wow

Comment (Deepwarrior) [reply axol/axol.py:2117]: a
eto drugoe

Comment (Deepwarrior) [reply axol/axol.py:2117]: dumal chto when next_step_handler otworkal, to pri try udalit him here bude exception
A you prosto timer otmenila

**Review by Deepwarrior — APPROVED** [review]

File: axol/axol.py (hunk 4 above)
Comment (UhiOfficial) [reply axol/axol.py:2117]: Cho

**Pull request merged**